manager/src/main/java/com/bakdata/quick/manager/k8s/resource/QuickResources.java

```diff
@@ -50,4 +50,11 @@ enum ResourcePrefix {
             this.prefix = deploymentPrefix;
         }
     }
+
+    /**
+     * A method for retrieving the name of resources (without prefix).
```

Comment:
I'm not the biggest fan of starting a comment with `A/This method/class...` because it doesn't add any information.

```suggestion
     * Retrieves the name of resources (without prefix).
```